tests/e2e/global.teardown.ts

```
@@ -0,0 +1,19 @@
+import { test as teardown } from "@playwright/test";
+import { exec } from "child_process";
+
+teardown("shut down docker compose", async ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)